content/development-guide/api/command.md

```diff
@@ -21,38 +20,35 @@ namespace SamplePlugin
 
 		public void Execute (ICommandContext context)
 		{
-			context.User.SendMessage ("The sample command says hi!", System.Drawing.Color.Yellow);
+			context.User.SendMessage ("The sample command says hi!", Rocket.API.Drawing.Color.Yellow);
 		}
 	}
 }
 ```
 
-**Name**: 			The commands name (e.g. `rocket`, `buy`, etc)
-**Aliases**:		A list of alternative for the name 
-**Summary**:		A short summary of the commands function
-**Description**:	A long description entailing the commands functionality, Not Required
-**Permission**:		The permission the player needs to use the command. Recommended to keep null as it defaults to PluginName.CommandName
-**Syntax**:			A syntax string for the command `[]` = *optional* and `<>` = *required* (e.g. [steamId] <itemId>)
-**ChildCommands**:	A list of child commands (e.g. `shop sell`, `shop buy` : `buy` and `sell` are child commands of `shop`)
-**SupportsUser**:	A bool to check if a user passed through is supported for this command
-**Execute**:		Where your code gets executed
+**Name:**		The commands name (e.g. `rocket`, `buy`, etc).
+**Aliases:**		A list of alternative for the name, not required.
+**Summary:**		A short summary of the commands function.
+**Description:**	A long description entailing the commands functionality, not required, not required.
```

Comment:
not required, not required, not required, not required, not required, not required

Reply:
I just meant `can be null or empty` with `not required`. Well, I was just did that because new developers should know which value can be null or empty. They can see the definiton of value by locating cursor on them in Visual Studio, though. But like this, definiton of them look more detailed in Website. They are like `This property can return null.` or `This property will never return null.` in source code, so why not in Website?